subsystem_controllers/config/guidance_controller_config.yaml

```diff
@@ -51,6 +51,6 @@
       - /guidance/plugins/stop_and_wait_plugin
       - /guidance/plugins/sci_strategic_plugin
       - /guidance/plugins/stop_controlled_intersection_tactical_plugin
-      - /guidance/plugins/platoon_strategic_ihp
+      - /guidance/plugins/platooning_strategic_ihp
```

Comment:
Comment not specific for here but there are still references to `platoon_strategic_ihp`

Reply:
I actually just reverted the chagnes for this plugin and let Anish handle it fully in his PR